DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "Template type is not one of [sms, email, letter]"`<br>`}]`|Contact the Notify team|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Error: Your system clock must be accurate to within 30 seconds"`<br>`}]`|Check your system clock|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: signature, api token not found"`<br>`}]`|Use the correct API key. Refer to [API keys](/#api-keys) for more information|
 
-</details>
+# Get a template
 
+## Get a template by ID
 
-#### Response
+### Method
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+This will return the latest version of the template.
 
-```Ruby
-template.id         # => uuid for the template
-template.type       # => type of template one of email|sms|letter
-template.created_at # => date and time the template was created
-template.updated_at # => date and time the template was last updated, may be null if version 1
-template.created_by # => email address of the person that created the template
-template.version    # => version of the template
-template.body       # => content of the template
-template.subject    # => subject for email templates, will be empty for other template types
+```ruby
+response = client.get_template_by_id(id)
 ```
 
-Otherwise the client will raise a `Notifications::Client::RequestError`.
+### Arguments
 
-</details>
+#### id (required)
 
+The ID of the template. You can find this by signing into GOV.UK Notify and going to the __Templates__ page.
 
-#### Arguments
+```
+id:'f33517ff-2a88-4f6e-b855-c550268ce08a'
+```
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+### Response
 
-|`error.code`|`error.message`|
-|:---|:---|
-|`404`|`[{`<br>`"error": "NoResultFound",`<br>`"message": "No result found"`<br>`}]`|
-|`404`|`[{`<br>`"error": "ValidationError",`<br>`"message": "is is not a valid UUID"`<br>`}]`|
+If the request to the client is successful, the client will return a `Notifications::Client::Template` object. This object is named `response` in the method example.
+
+You can then call different methods on this object to return the requested information:
 
-##### `id`
-The template id is visible on the template page in the application.
+|Method|Information|Type|
+|:---|:---|:---|
+|`response.id`|Template UUID|String|
+|`response.type`|Template type (email/sms/letter)|String|
+|`response.created_at`|Date and time template created|String|
+|`response.updated_at`|Date and time template last updated (may be nil if version 1)|String|
+|`response.created_by`|Email address of person that created the template|String|
+|`response.version`|Template version|String|
+|`response.body`|Template content|String|
+|`response.subject`|Template subject (email only)|String|
```

Comment:
Letters have a subject too - for letters the subject is the main heading of the letter.